web/controllers/registration_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Yggdrasil.RegistrationController do
```

Comment:
Ah wasn't sure to what degree you meant. I was ok with it being RegistrationController, reads better than RegisterController but the url I think makes more sense being called register to the user.